squirrel.worker/src/main/java/org/dice_research/squirrel/analyzer/impl/html/scraper/HtmlScraper.java

```diff
@@ -324,15 +328,12 @@ private String replaceCommands(String s) {
     	
     	List<Node> listNodes = new ArrayList<Node>();
     	
-    	@SuppressWarnings("unused")
-		boolean useResource = false;
-    	
     	 try {
           	
-          	if(cssQuery.startsWith("l(")) {
```

Comment:
The bracket had to be there. Why did you remove it?

Reply:
Added it back. It was missed during resolving merge conflicts.